Source/Scene/Scene.js

```diff
@@ -434,11 +442,19 @@ define([
          */
         this.fxaa = false;
 
+        /**
+         * The time in milliseconds to wait before checking if the camera has not moved and fire the cameraMoveEnd event.
+         * @type {Number}
+         * @default 500.0
+         */
+        this.cameraEventWaitTime = 500.0;
```

Comment:
To reduce the surface area of the API, perhaps mark this `@private` until we know we need it?